bootstrap/bootstrap.py

```diff
@@ -356,6 +360,7 @@ def serve_forever(server):
                 "python3-venv",
                 "python3-pip",
                 "git",
+                "sudo",
```

Comment:
I assume `sudo` is installed in ubuntu by default, but needs to be installed in debian - is that correct? If so, let's add an inline comment about it!

Reply:
Sudo is. indeed, not installed in Debian by default. But adding it here is harmless if it is installed and fixes it for Debian.

Reply:
I think this is OK, even if it's pre-installed in Ubuntu this acts as documentation that sudo is required. It's also useful if someone uses a custom Ubuntu cloud image with minimal dependencies.

On the other hand https://tljh.jupyter.org/en/latest/install/custom-server.html#step-1-installing-the-littlest-jupyterhub mentions `sudo apt install python3 python3-dev git curl` is a pre-requisite so we should decide what is a prerequisite, and what should be managed by `bootstrap.py`

Reply:
You are right. But since I needed to modify the script and I never run scripts from the pipe to sudo I downloaded the script with wget and run it with python. If the sudo was not installed previously, the installation would fail. 
Note that, there is no harm in requiring installation of already installed component.

Reply:
OK, presumably you're running `bootstrap.py` as `root` then? That's why I asked for your workflow :smile: 

It looks like the sudo package is only required for some config files as far as the installer is concerned:
https://github.com/jupyterhub/the-littlest-jupyterhub/blob/0572976d25c18bc88d936f8bd8627bbfed069b53/tljh/installer.py#L147-L153

Reply:
Yes, I was running it from root. Sudo is required in ubuntu install and probably (?) during some admin tasks. Debian is not using it for admin, but it is fully supported there.